demisto_sdk/commands/find_dependencies/find_dependencies.py

```diff
@@ -291,6 +291,120 @@ def _collect_playbooks_dependencies(pack_playbooks, id_set):
 
         return dependencies_packs
 
+    @staticmethod
+    def _collect_layouts_dependencies(pack_layouts, id_set):
+        """
+        Collects layouts pack dependencies.
+
+        Args:
+            pack_layouts (list): collection of pack playbooks data.
+            id_set (dict): id set json.
+
+        Returns:
+            set: dependencies data that includes pack id and whether is mandatory or not.
+
+        """
+        dependencies_packs = set()
+
+        for layout in pack_layouts:
+            layout_data = next(iter(layout.values()))
```

Comment:
why do we need next and iter

Reply:
Its just an easy way to get the data of the layout.
Thats how it is done in the rest of the code, so I wanted to keep the structure aligned